staging/src/k8s.io/component-base/featuregate/registry_test.go

```diff
@@ -125,7 +125,7 @@ func TestVersionFlagOptionsWithMapping(t *testing.T) {
 		t.Errorf("wanted emulation version flag options to be: %s, got %s", expectedEmuVers, emuVers)
 	}
 	minCompVers := strings.Join(r.unsafeVersionFlagOptions(false), "\n")
-	expectedMinCompVers := "kube=1.30..1.31 (default=1.30)\ntest=2.7..2.8 (default=2.7)"
+	expectedMinCompVers := "test=2.7..2.8 (default=2.7)"
```

Comment:
it's surprising the kube portion dropped from the message... is that intentional?

Reply:
It was not intentional. This has been updated: https://github.com/kubernetes/kubernetes/pull/128279#discussion_r1894241284